src/libraries/Microsoft.Bcl.Memory/src/Microsoft.Bcl.Memory.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;$(NetFrameworkMinimum);$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Should we also have NetCoreAppPrevious here? Presumably this should work on .NET 8 using something newer than the netstandard2.0 build, right?

Reply:
It'll use the `netstandard2.1` build there, which should be fine from type location perspective (unless it's not for URL impl).

Reply:
> (unless it's not for URL impl).

The Base64Url implementation is new to .NET 9 and it uses Vector128/256/512 that aren't in netstandard2.1. Ideally someone using this package on .NET 8 would get the faster implementation.

Reply:
I am not sure I follow. `Base64Url ` will be poly filled for ns2.1. So, running on .NET 8.0 will get this poly fill implementation. How .NET 8.0 will be faster?

Reply:
> How will .NET 8.0 get faster?

Because on .NET 8 we could use the implementation that uses Vector128/Vector256/Vector512. The netstandard2.1 implementation can't.

Reply:
Does this implementation is ready? I am asking to know if we can add the 8.0 target when we have it.  or can we copy the 9.0 implementation? Any reason we didn't do this in the first place?

Reply:
> Any reason we didn't do this in the first place?

None that I can think of, other than it was just a miss.

> I am asking to know if we can add the 8.0 target when we have it. or can we copy the 9.0 implementation?

The implementation relies on ifdefs to pick up the right implementation. Given that the implementation uses `#if NET`, I expect it'll "just work". It's possible there are some .NET 9-only things currently in those if blocks, but if there are any they should be easy to deal with.

Reply:
I'll give it a try.

Reply:
Looking more discovered that the implementation in .NET 9.0 uses internal members of the public `Ascii` class which need more work to refactor the code and create a suitable .NET 8.0 implementation. I have created the issue https://github.com/dotnet/runtime/issues/104185 tracking that.